packages/aws-cdk-lib/aws-cognito/lib/user-pool-idps/saml.ts

```diff
@@ -99,6 +124,9 @@ export class UserPoolIdentityProviderSaml extends UserPoolIdentityProviderBase {
         IDPSignout: props.idpSignout ?? false,
         MetadataURL: metadataType === UserPoolIdentityProviderSamlMetadataType.URL ? metadataContent : undefined,
         MetadataFile: metadataType === UserPoolIdentityProviderSamlMetadataType.FILE ? metadataContent : undefined,
+        EncryptedResponses: props.encryptedResponses ?? undefined,
+        RequestSigningAlgorithm: props.signingRequests ? 'rsa-sha256' : undefined,
```

Comment:
I could not find any information about what algorithm is acceptable for `RequestSigningAlgorithm` without `rsa-sha256`.

https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpoolidentityprovider.html#aws-resource-cognito-userpoolidentityprovider-properties
> SAML
> Create or update request with Metadata URL: "ProviderDetails": { "IDPInit": "true", "IDPSignout": "true", "EncryptedResponses" : "true", "MetadataURL": "https://auth.example.com/sso/saml/metadata", "RequestSigningAlgorithm": "rsa-sha256" }

I defined `signingRequest` as boolean but it may be better to define `SigningAlgorithm` enum and use it.
I would like to hear the reviewer's opinion.

Reply:
I'm also unable to find documentation regarding other availabilities, maybe someone from AWS can clear it up if they have internal docs regarding this.

Regardless, even if `rsa-sha256` is the only current available signing algorithm, adding an enum right away gives us more extensibility if and when other algorithms are made available. The boolean would become redundant.

Reply:
@nmussy Thank you for your opinion!!
I will create a new enum and remove Boolean definition.

Reply:
This would have been my one comment on this PR. Great suggestion.